src/core/AutoRest/Simplify/CSharpSimplifier.cs

```diff
@@ -92,9 +92,20 @@ private static MetadataReference Mscorlib
                         // get rid of any BOMs 
                         .Trim('\x00EF', '\x00BB', '\x00BF', '\uFEFF', '\u200B');
 
-                    // Write out the files back to their original location
-                    var output = Path.Combine(Settings.Instance.FileSystem.CurrentDirectory, document.Name);
-                    Settings.Instance.FileSystem.WriteFile(output, text);
+
+                    // special cases the simplifier can't handle.
+                    text = text.
```

Comment:
Yeah, that would not be likely. 
As it is, this is somewhat temporary until we get to write a better simplifier that's more agressive, but not until after dotnet cli RTM ships.